db/schema.rb

```diff
@@ -506,7 +505,7 @@
     t.integer "minimum_level", null: false
   end
 
-  create_table "course_condition_surveys", id: :serial, force: :cascade do |t|
+  create_table "course_condition_surveys", force: :cascade do |t|
```

Comment:
Your `schema.rb` is changing things that isn't affected by migration?

Reply:
done